driver/collector/postgres_collector.py

```diff
@@ -301,6 +301,26 @@
 AND schemaname !~ '^pg_toast';
 """
 
+QUERY_INDEX_SCHEMA_COLUMN_NAMES = """
```

Comment:
Please feel free to just review this query +  the `collect_schema` method below for this file. The other parts are format changes.